src/js/mods/modloader.js

```diff
@@ -141,9 +161,26 @@ export class ModLoader {
         LOG.log("hook:init", this.app, this.app.storage);
         this.exposeExports();
 
-        // TODO: Make use of the passed file name, or wait for ModV2
-        let mods = await ipcRenderer.invoke("get-mods");
-        mods = mods.map(mod => mod.source);
+        // TODO: Make use of the passed file name
+        const decoder = new TextDecoder()
+        /**
+         * @type {string[]}
+         */
+        let mods = (await ipcRenderer.invoke("get-mods")).map(( /**@type {IPCMod} */mod) => {
+            const contents = mod.contents.contents;
```

Comment:
maybe using unique names for the three instances of `.contents` would be better?